reproducible/build.Dockerfile

```
@@ -1,15 +1,17 @@
 # A builder for images for verifiable/reproducible builds.
-# The `source_image` is meant to be a published Rust image, such as rust:1.70.
+# The `source_image` is meant to be a published Rust image, such as rust:1.90.
 #
 # This image only adds a script `run-copy.sh` that is used by cargo-concordium
 # to copy data in and out of the container. The image also adds
 # wasm32-unknown-unknown target.
 
-ARG source_image
+ARG source_image=rust:1.86.0
```

I would prefer if the flow fails instead of having a default source image here.